packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -1,31 +1,121 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
+import QuickLinks from './components/quick-links/QuickLinks';
+import NavItems from './components/nav-items/NavItems';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
- /*
- * Content to be displayed as the name
- */
-  name: PropTypes.string,
+  test: PropTypes.string,
+  help: PropTypes.string,
+  isMobileNavOpen: PropTypes.bool, // this is proof of concept. should we keep this?
```

Comment:
This should probably exist as state, not a property as it will definitely change.